src/react/AppProvider.jsx

```diff
@@ -61,8 +62,28 @@ export default function AppProvider({ store, children }) {
     setLocale(getLocale());
   });
 
+  useEffect(() => {
+    if (config.THEME_OVERRIDE_URL) {
```

Comment:
todo: correctly handle dynamic `THEME_OVERRIDE_URL` change (e.g., don't create new `link`). Although I don't think it can change during runtime, should still consider such case.